optimize/backend/pom.xml

```diff
@@ -12,12 +14,11 @@
   <name>Optimize Backend</name>
 
   <properties>
-    <jersey.version>3.1.3</jersey.version>
     <spring.security.version>6.3.4</spring.security.version>
-    <nimbus.jose.jwt.version>9.41.2</nimbus.jose.jwt.version>
-    <it.test.excludedGroups />
-    <it.test.includedGroups />
-    <test.forkCount />
+    <it.test.excludedGroups/>
+    <it.test.includedGroups/>
+    <test.forkCount/>
+    <version.tomcat>10.1.30</version.tomcat>
```

Comment:
Why use `10.1.30` when `10.1.33` is the latest version?

Reply:
if I remember well, I simply followed the suggestion of the IDE. I'll try with the latest version